tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(STL_SOURCE_DIR "${msvc_standard_libraries_SOURCE_DIR}" CACHE PATH
+    "Location of the stl source tree")
+set(STL_TEST_UTILS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/utils" CACHE PATH
+    "Location of the stl test utilities")
+set(STL_BUILD_ROOT "${msvc_standard_libraries_BINARY_DIR}/out" CACHE PATH

```

Comment:
don't do this, for starters this is probably wrong, because vs creates out/{config-name}/... but also just use ${PROJECT_BINARY_DIR} instead

Reply:
On my machine VS creates: `C:\Users\CUBEZAUL\source\Repos\STL\out\build\x64`
The name is misleading as it isn't the build root but it is where all the `bin`, `lib` and `inc` folders live.